### PR TITLE
DF-513:Make record attributes consistent with API response

### PR DESCRIPTION
**Pull request by TNA-Allan**
Related ticket(s):
https://national-archives.atlassian.net/browse/DF-513

## About these changes

Some record attributes are inconsistent with the API response attribute. This change is to make them consistent and apply them to the templates. 
**Making them consistent enables easier debug and maintenance.**

- created_date
- access_condition
- summary_title
- delivery_option

## How to check these changes

As mentioned in the ticket.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer.
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [x] Waited for all CI jobs to pass before requesting a review.
- [x] Added/updated tests and documentation where relevant.

## For Reviewer/Developer

- Before merging PR

    `main` branch :

        - check PR merge tile begins with `release/<major.minor.patch>:<default/custom description>`
        - check PR merge tile begins with ticket number Ex `DF-XXX: Ticket name / short description`
        - select `Create a merge commit`

    `develop` branch:

        - check if merges into base branch are `not to be kept On Hold` and then proceed.
        - check PR merge tile begins with ticket number Ex `DF-XXX: Ticket name / short description`
        - select `Squash and merge`

- After merging PR:

  - check and arrange to deploy on platform-sh.
  - update status on JIRA ticket on deployment.
  - update slack on deployment progress of the JIRA ticket once deployed.

### CODE REVIEW

File: etna/records/field_labels.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # format <record_field_name> : <human readable label name>
 FIELD_LABELS = {
-    "title": "Title",
+    "summary_title": "Summary title",
```
Comment (ababic) [inline etna/records/field_labels.py:3]: Since the records have both, maybe we should keep both here?

File: etna/records/models.py [hunk 2]
```diff
@@ -163,7 +163,7 @@ def has_url(self) -> bool:
             return True
 
     @cached_property
-    def title(self) -> str:
+    def summary_title(self) -> str:
```
Comment (ababic) [inline etna/records/models.py:166]: Is there also a 'title' field that we can pull out from somewhere?

**Review by ababic — COMMENTED** [review]
These looks great, thanks! I just had a couple of questions on whether we should be replacing `title` completely, or whether we just need to pull out a different value.

**Comment by TNA-Allan** [comment]
> These looks great, thanks! I just had a couple of questions on whether we should be replacing title completely, or whether we just need to pull out a different value
Any inputs from API which was not brought into the details page will be approved from here https://national-archives.atlassian.net/browse/DF-417, in this case `Title`

File: etna/records/models.py (hunk 2 above)
Comment (TNA-Allan) [reply etna/records/models.py:166]: Yes, but needs approval https://national-archives.atlassian.net/browse/DF-417

File: etna/records/field_labels.py (hunk 1 above)
Comment (TNA-Allan) [reply etna/records/field_labels.py:3]: Bringing in Title needs approval https://national-archives.atlassian.net/browse/DF-417

**Review by ababic — APPROVED** [review]
Thanks for confirming Allan. In that case, 👍

**Pull request merged**